ts/DataGrid/Actions/ColumnsResizer.ts

```diff
@@ -140,6 +143,31 @@ class ColumnsResizer {
         column.width = vp.getRatioFromWidth(newLeftW);
         nextColumn.width = vp.getRatioFromWidth(newRightW);
     }
+    
+    /**
+     * Render the drag handle for resizing columns.
+     */
+    public renderColumnDragHandles(column: Column, cell: Cell): void {
```

Comment:
Shouldn't the handle be through the whole height of the grouped column?
![image](https://github.com/user-attachments/assets/980e795b-f659-4fba-9204-2e0af92cb30b)

Reply:
I render the resizer for the last columns only. So IMO it's subject for another discussion.

Reply:
Now I see:
![image](https://github.com/user-attachments/assets/293b9f79-6bbe-47d4-a93e-8fafee2af729)
and:
![image](https://github.com/user-attachments/assets/324e59f5-bea3-41bf-a946-aeadcca515fd)


We should be consistent with placing it.

Reply:
We also discussed it with Dawid interally, It's related to row spans etc., requiring extra algorithms for positioning elements inside. Let's leave it like it is for now.